index.html

```diff
@@ -1,59 +1,425 @@
+
+      <br><br>
+      <div class="imagebox3">
+        <img src="image/Img Plaholder.png" alt="placeholder" >
+      
+      </div>
+      <div class="storiescontainer storiescontainer3">
+      <div class="head3">
+        <h1>Multi-Post Stories Gain+Glory</h1>
+      </div>
+      <div class="insideautolanguage">
+      <div class="language">
+        <ul>
+          <li><a href="#">Ruby on rails</a></li>
+      
+              <li><a href="#">CSS</a></li>
+              <li><a href="#">Javascript</a></li>
+              <li><a href="#">HTML</a></li>
+        </ul>
+      
+      </div>
+      </div>
+      <div class="insideautobutton">
+      <div class="seeproject">
+        <button>See Project</button>
+      </div>
+      </div>
+      </div>
+        </div>
+      </div>
+
+<br><br>
+        <div class="imagebox4">
+          <img src="image/Img Plaholder.png" alt="placeholder" >
+        
+        </div>
+        <div class="storiescontainer storiescontainer4">
+        <div class="head3">
+          <h1>Multi-Post Stories Gain+Glory</h1>
+        </div>
+        <div class="insideautolanguage">
+        <div class="language">
+          <ul>
+            <li><a href="#">Ruby on rails</a></li>
+        
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">HTML</a></li>
+          </ul>
+        
+        </div>
+        </div>
+        <div class="insideautobutton">
+        <div class="seeproject">
+          <button>See Project</button>
+        </div>
+        </div>
+        </div>
           </div>
+        </div>
+
+<br><br>
+          <div class="imagebox5">
+            <img src="image/Img Plaholder.png" alt="placeholder" >
           
-          <div class="mobilemenu">
-          <i class="fa-sharp fa-solid fa-bars"></i>  
-           </div>
-          </nav>
+          </div>
+          <div class="storiescontainer storiescontainer5">
+          <div class="head3">
+            <h1>Multi-Post Stories Gain+Glory</h1>
+          </div>
+          <div class="insideautolanguage">
+          <div class="language">
+            <ul>
+              <li><a href="#">Ruby on rails</a></li>
           
+                  <li><a href="#">CSS</a></li>
+                  <li><a href="#">Javascript</a></li>
+                  <li><a href="#">HTML</a></li>
+            </ul>
           
-        <nav class="desktop-nav">
-            <ul class="desktop-navmenu">
-                <li><a href ="#Portfolio">Portfolio</a></li>
-                <li><a href="#About">About</a></li>
-                <li><a href="#Contact">Contact</a></li>
+          </div>
+          </div>
+          <div class="insideautobutton">
+          <div class="seeproject">
+            <button>See Project</button>
+          </div>
+          </div>
+          </div>
+            </div>
+          </div>
+</section>
 
-            </ul>
-        </nav>
-    </header>
-       
+<!--aboutme-->
```

Comment:
- [ ] Please make sure here is perfectly aligned and following Figma style guidelines.

## Figma Design
![image](https://user-images.githubusercontent.com/92215878/220712681-bedf0d2d-2170-4f7e-a3b0-9578e1df3fa7.png)


## Your Design
![image](https://user-images.githubusercontent.com/92215878/220712519-9a446933-09ab-4c8a-8508-dfeafdabe5c8.png)